python/guide/module/gluon/init.md

```diff
@@ -161,3 +170,127 @@ net.add(nn.Dense(10, in_units=256))
 
 net.initialize(init=MyInit())
 ```
+
+## Parameter Initialization
+
+Now that we know how to access the parameters, let's look at how to initialize
+them properly. We discussed the need for
+[Initialization](../chapter_deep-learning-basics/numerical-stability-and-init.md)
```

Comment:
The link needs to be adjusted.

Reply:
Removed sentence as it was D2L specific.